recipe/data.go

```diff
@@ -3231,7 +3231,7 @@ var Suicides = Response{
 	OutputInstances: []instance{
 		{
 			DatasetID: "suicides-in-the-uk",
-			Editions:  []string{"time-series"},
+			Editions:  []string{"2012", "2013", "2014", "2015", "2016", "2017", "2018", "2019", "2020", "time-series"},
```

Comment:
If we are adding specific years now is `time-series` still required? Only asking because I don't know.

Reply:
Yes, because it already exists we might need to add corrections to the data already published in that edition